README.md

```diff
@@ -1,2 +1,56 @@
+
+If your HIP requires images, the image files should be included in a subdirectory of the `assets` folder for that HIP as follows: `assets/hip-N` (where **N** is to be replaced with the HIP number). When linking to an image in the HIP, use relative links such as `../assets/hip-1/image.png`.
+
+Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft HIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your HIP contains either your GitHub username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
+
+When you believe your HIP is mature and ready to progress past the draft phase, you should do one of two things:
+
+ - **For a Standards Track HIP of type Core**, ask to have your issue added to [the agenda of an upcoming Handshake Developer meeting](Link to Meeting Schedule), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the HIP editors will update the state of your HIP to 'Accepted'.
+ - **For all other HIPs**, open a PR changing the state of your HIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the HIP - they may close the PR and request that you fix the issues in the draft before trying again.
+
+# HIP Status Terms
+
+* **Draft** - an HIP that is undergoing rapid iteration and changes.
+* **Last Call** - an HIP that is done with its initial iteration and ready for review by a wide audience.
+* **Accepted** - a core HIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Handshake Client Developers to decide whether to encode an HIP into their clients as part of a hard fork is not part of the HIP process. If such a decision is made, the HIP will move to final.
+* **Final (Non-Core)** - an HIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
+* **Final (Core)** - an HIP that the Handshake Client Developers have decided to implement and release in a future hard fork or has already been released in a hard fork. 
+
+# Preferred Citation Format
+
+The canonical URL for HIP that has achieved draft status at any point is at the Handshake-Org HIPs. For example, the canonical URL for HIP-1 is (Link to HIP-001)
```

Comment:
link is missing. Also my preference is capitalization like "handshake-org" since thats the actual style of the github organization.